net/etesync-server/files/80_setup-etesync-server

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+[ "${PKG_UPGRADE}" = "1" ] && /etc/init.d/etesync-server stop
+
+cd /usr/share/etesync-server || exit 1
+
+if [ ! -f "db.sqlite3" ]
```

Comment:
Might be cleaner to do

[ -f "db.sqlite3" ] || { }

Reply:
Changed it ...